wapi/auth.py

```diff
@@ -37,7 +37,7 @@ def validate_auth(self):
         # Prevent loop
         if self.revalidating:
             return
-        if self.valid_until is None or time.time() > self.valid_until:
+        if self.valid_until is None or time.time() > self.valid_until - 60:
```

Comment:
The check is useful, but maybe not use a fixed offset?  In other projects, I have done it by multiplying `rsp['expires_in']` by 0.9 when setting `valid_until`, could that be an approach here as well?